lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -297,28 +297,28 @@ class ClassMetadataInfo implements ClassMetadata
     /**
      * READ-ONLY: The names of the parent classes (ancestors).
      *
-     * @var array
+     * @psalm-var list<class-string>
      */
     public $parentClasses = [];
 
     /**
      * READ-ONLY: The names of all subclasses (descendants).
      *
-     * @var array
+     * @psalm-var list<class-string>
      */
     public $subClasses = [];
 
     /**
      * READ-ONLY: The names of all embedded classes based on properties.
      *
-     * @var array
+     * @psalm-var array<string, mixed[]>
      */
     public $embeddedClasses = [];
 
     /**
      * READ-ONLY: The named queries allowed to be called directly from Repository.
      *
-     * @var array
+     * @psalm-var array<string, array<string, mixed>>
```

Comment:
```suggestion
     * @psalm-var array<string, array{name: string, query: string, dql: string}>
```

inferred from `\Doctrine\ORM\Mapping\ClassMetadataInfo::getNamedQuery`
inferred from `\Doctrine\ORM\Mapping\ClassMetadataInfo::addNamedQuery`